### DISCUSSION TITLE
Help needed for Lesson 7 Challenge!!

### DISCUSSION THREAD

**Discussion by subhradeepsamantray** [Cyfrin/foundry-full-course-cu#725] [Q&A]
Hello All,

I have been trying to solve the Lesson 7 challenge but I am not finding any clue how to approach the problem. If anyone can be me some hints on how to solve this problem, please respond. I am not able to find the proper commands to use to reach find what is stored in the storage slot

**Comment by shikhar229169** [comment]
This is the hint:

https://book.getfoundry.sh/reference/cast/cast-storage

**Reply by subhradeepsamantray** [reply]
Hello Shikhar, thanks for the help. I have one question -> How to get the contract address ? Will I need to deploy the contract in my local system ?

**Reply by subhradeepsamantray** [reply]
Hello Shikhar, thanks for the help , I have got the answer now 🙂. I figured out that I need to take the contract address of the challenge and the rpc url of a sepolia ETH testnet node to make the command work. Thanks mate 👍
